FRW/Modules/Buy/BuyProvderView.swift

```diff
@@ -37,6 +38,7 @@ struct BuyProvderView: View {
 
             if LocalUserDefaults.shared.flowNetwork == .mainnet {
                 Button {
+                    EventTrack.General.rampClick(source: "coinbase")
```

Comment:
nit: it would be better to collect all parameter names and values under a single enumeration, just to prevent mistypes that sooner or later might happen.